js/NewsChannelOperations.js

```diff
@@ -0,0 +1,44 @@
+function SaveEmail(){
+  var emailAddresss= document.getElementById('txtEmailAddress').value;
+  if (ValidateEmail(emailAddresss)){
```

Comment:
Show a success/error message below the email text box with green and red colors. Currently the user does not know whether email is saved or not.

Reply:
updated code to show information message to the user